src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -102,6 +108,21 @@ SHOULD be present: For consistency between studies and institutions, we
 encourage users to extract the values of these fields from the actual raw data.
 Whenever possible, please avoid using ad hoc wording.
 
+{{ MACROS___make_metadata_table(
+   {
+      "InstitutionName": "RECOMMENDED",
+      "InstitutionAddress": "RECOMMENDED",
+      "Manufacturer": ("RECOMMENDED", 'For example, `"Biosemi"`, `"Brain Products"`, or `"Neuroscan"`.'),
```

Comment:
```suggestion
      "Manufacturer": ("RECOMMENDED", 'For example, `"TDT, Blackrock"`.'),
```

Reply:
Should this be `"TDT, Blackrock"` or `"TDT"`, `"Blackrock"`?

Reply:
I have no idea. @dorahermes can you help us what this example should be?

Reply:
Addressed in 34d9435.